MatrixSDK/Crypto/Algorithms/RoomEvent/MXRoomEventDecryption.swift

```diff
@@ -0,0 +1,218 @@
+// 
+// Copyright 2022 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+#if DEBUG
```

Comment:
Can you clarify why this type needs to exist just for debug builds?

Reply:
Yea the entire `MatrixSDKCrypto` pod is only available in `DEBUG` model (see [here](https://github.com/matrix-org/matrix-ios-sdk/blob/develop/Podfile#L19)), which is mostly for safety whilst Crypto V2 is being developed. This way we can keep merging changes to develop without worrying about accidentally shipping it to production.